base/bin/migration.mustache

```diff
@@ -2,9 +2,9 @@ const fs = require('fs');
 const mustache = require('mustache');
 
 const now = new Date().toISOString();
-const migrationName = now + '_' + process.argv[2];
+const migrationName = `${now}_${process.argv[2]}`;
```

Comment:
Oh, and this has actually already been merged. I've just been talking to myself in a darkened room. Nice.